drivers/crypto/crypto_mtls_shim.c

```diff
@@ -21,6 +21,7 @@
 #endif /* CONFIG_MBEDTLS_CFG_FILE */
 
 #include <mbedtls/ccm.h>
+#include <mbedtls/gcm.h>
```

Comment:
is this header available when mbedTLS is built without gcm support ? Shouldn't you put this include under ifdef ?

Reply:
Yes, it's available - but you are right, it's probably better to include `gcm.h` conditionally. That would mean that the `mbedtls_gcm_context` in the session would also have to be surrounded by an `ifdef`, though. Will change it.